clonevirtualenv.py

```diff
@@ -298,7 +305,7 @@ def main():
     try:
         old_dir, new_dir = args
     except ValueError:
-        print("virtualenv-clone {}".format(__version__))
+        print("virtualenv-clone %s" % __version__)
```

Comment:
This change shouldn't be required.

Reply:
Same reason as the above "2.6 failing" story